packages/tasit-action/src/contract/Action.js

```diff
@@ -6,32 +6,67 @@ import ConfigLoader from "../ConfigLoader";
 //  and/or MetaTxAction subclasses
 export class Action extends Subscription {
   #provider;
+  #signer;
```

Comment:
How does this end up being used?

Reply:
I guess it's often enough that the signer isn't the sender in our lib with contract-based accounts and meta-tx that we shouldn't default this like we used to.

Reply:
Hmm, the `Action` is receiving the `rawTx` and the `signer`, on the `send()` function (it's an alias for `signAndSend()` the `rawTx` is being decorated (gasParam and nonce), sign and sent.
We can have the action receiving the `signedTx` instead but will need to handle with gas and nonce outside?
A question: Shouldn't all transactions (even the meta-tx) be signed before broadcasted?

Reply:
Moving thread here https://github.com/tasitlabs/TasitSDK/pull/357/files/ea53439d35f7f5e022c6d3e1e4f09d6b4b02b617#r279128314

Reply:
Resolving this since the thread was moved